mergedSchema.js

```diff
@@ -0,0 +1,82 @@
+import { mergeSchemas, introspectSchema, makeRemoteExecutableSchema } from "graphql-tools"
+import { ApolloLink } from "apollo-link"
+import { createHttpLink } from "apollo-link-http"
+import { setContext } from "apollo-link-context"
+import fetch from "node-fetch"
+
+import config from "./config"
+import localSchema from "./schema"
+
+function createConvectionLink() {
+  const httpLink = createHttpLink({
+    fetch,
+    uri: `${process.env.CONVECTION_API_BASE}/graphql`,
+  })
+
+  const middlewareLink = new ApolloLink((operation, forward) => forward(operation))
+
+  const authMiddleware = setContext((_request, context) => {
+    const loaders = context.graphqlContext && context.graphqlContext.res.locals.dataLoaders
+    const tokenLoader = loaders && loaders.convectionTokenLoader
+
+    // If a token loader exists for Convection (i.e. this is an authenticated request), use that token to make
+    // authenticated requests to Convection.
+    if (tokenLoader) {
+      return tokenLoader().then(({ token }) => {
+        return {
+          ...context.headers,
```

Comment:
Note to self, this is wrong, it needs to add all headers to the below object instead.